### ISSUE TITLE
npm install: 404 Not Found: tcomb-form@0.9.19

### ISSUE THREAD

**Issue by iztokf** [gcanti/tcomb-form#416]
### Version

- tcomb-form v0.9.19?
- react v16.3.2.?

### Stack trace and console log

14275 verbose stack Error: 404 Not Found: tcomb-form@0.9.19
14275 verbose stack     at fetch.then.res (/usr/local/lib/node_modules/npm/node_modules/pacote/lib/fetchers/registry/fetch.js:42:19)
14275 verbose stack     at tryCatcher (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/util.js:16:23)
14275 verbose stack     at Promise._settlePromiseFromHandler (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:512:31)
14275 verbose stack     at Promise._settlePromise (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:569:18)
14275 verbose stack     at Promise._settlePromise0 (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:614:10)
14275 verbose stack     at Promise._settlePromises (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:693:18)
14275 verbose stack     at Async._drainQueue (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/async.js:133:16)
14275 verbose stack     at Async._drainQueues (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/async.js:143:10)
14275 verbose stack     at Immediate.Async.drainQueues [as _onImmediate] (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/async.js:17:14)
14275 verbose stack     at runCallback (timers.js:696:18)
14275 verbose stack     at tryOnImmediate (timers.js:667:5)
14275 verbose stack     at processImmediate (timers.js:649:5)
14277 verbose Linux 3.13.0-149-generic
14278 verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
14279 verbose node v10.2.1
14280 verbose npm  v6.1.0
14281 error code E404
14282 error 404 Not Found: tcomb-form@0.9.19
14283 verbose exit [ 1, true ]

**Comment by callmephilip** [comment]
Seeing the same with yarn:

```
error An unexpected error occurred: "https://registry.yarnpkg.com/tcomb-form/-/tcomb-form-0.9.19.tgz: Request failed \"404 Not Found\""
```

**Comment by callmephilip** [comment]
Seems very similar to this: https://github.com/npm/registry/issues/292

**Comment by callmephilip** [comment]
Fixed this temporarily by pointing the `tcomb-form` dep to the right tarball url:

```
"tcomb-form": "https://registry.yarnpkg.com/tcomb-form/tcomb-form-0.9.19.tgz"
```

**Comment by gcanti** [comment]
@iztokf @callmephilip re-published (as `tcomb-form@0.9.20`), should be fixed now
